deliver/lib/deliver/detect_values.rb

```diff
@@ -67,5 +69,17 @@ def find_platform(options)
         options[:platform] = 'osx'
       end
     end
+
+    def verify_languages!(options)
+      languages = options[:languages]
+      return unless languages
+
+      all_languages = Spaceship::Tunes.client.available_languages
+      diff = languages - all_languages
```

Comment:
Yeah, I didn't mean case insensitive usage, just for the compare operation :) I know ITC is picky about casing 🙄